run.py

```diff
@@ -2,5 +2,27 @@
 app = create_app()
 app.app_context().push()
 
+import os
+import rollbar
+import rollbar.contrib.flask
+from flask import got_request_exception
+
+@app.before_first_request
+def init_rollbar():
+    """init rollbar module"""
+    rollbar.init(
+        # access token
+        'ce78c3c226a9415d8fcf617cc993188c',
+        # environment name
+        'production',
```

Comment:
add access_token and environment in config.py file and then import from config and use it here